### PR TITLE
Issue #989 Setting a fixed version for cheggaaa which provides progre…

### PR DESCRIPTION
**Pull request by hferentschik**
…ss bar

Addresses issue #989

### CODE REVIEW

File: glide.yaml [hunk 1]
```diff
@@ -55,6 +55,7 @@ import:
   - ssh
 - package: golang.org/x/oauth2
 - package: gopkg.in/cheggaaa/pb.v1
+  version: v1.0.13
```
Comment (LalatenduMohanty) [inline glide.yaml]: 👍

**Comment by gbraad** [comment]
LGTM

**Comment by gbraad** [comment]
retest please

**Comment by gbraad** [comment]
CircleCI is not starting the build

**Pull request merged**